deployments/cluster/educator.nix

```diff
@@ -20,6 +20,18 @@ in
     4040        # Educator HTTP API
   ];
 
+  services.postgresql = {
+    enable = true;
+    authentication = ''
+      host all postgres 127.0.0.1/32 trust
+      host all postgres ::1/128      trust
+    '';
+    initialScript =
+      let
+        inherit (pkgs.disciplina-educator) name compiler;
+      in "${pkgs.disciplina-data}/share/${compiler.name}/${compiler.system}-${compiler.name}/${name}/database/schema.sql";
+  };
```

Comment:
Is it needed to execute [the schema creation](https://github.com/DisciplinaOU/disciplina/blob/master/educator/database/schema.sql) later upon node launch?

Reply:
I set `initialScript` to the path to your `schema.sql`. AFAIU this script is executed exactly once on the very first start of the postgres service.
(If there will be schema migrations, they’ll have to be handled separately.)

Reply:
Note that if the postgres service has already been started before, even for unrelated reasons, `initialScript` will not be run. It is not reliable.

Reply:
Actually, we already execute this schema from the code, so this is not needed unless you have a strong reason to do this here.

Reply:
That's a much better place, it should be removed here.

Reply:
Oh, ok, I misread “The database should exist upon the executable launch” in the original ticket description.